transports/janus_pfunix.c

```diff
@@ -441,6 +443,22 @@ void janus_pfunix_session_over(janus_transport_session *transport, guint64 sessi
 	janus_mutex_unlock(&clients_mutex);
 }
 
+void janus_pfunix_session_claimed(janus_transport_session *transport, guint64 session_id) {
+	/* Someone else has claimed the session: close the connection */
```

Comment:
Apart from the fact that the transport instance here would be the new one, and not the old, why close the previous transport connection just because a session has been claimed by another transport? I see legitimate cases where a different transport might claim a session and leave the old transport alive (e.g., it's still handling other sessions). I'd only use the reclaim stuff for handing sessions over.

I guess this was motivated by how session timeouts currently do close connections in `session_over`, which is why I think we should envisage the claim scenario as different.